api/build/create.go

```diff
@@ -129,6 +131,13 @@ func CreateBuild(c *gin.Context) {
 		queue.FromContext(c),
 	)
 
+	// check if build was skipped
+	if strings.EqualFold(item.Build.GetStatus(), constants.StatusSkipped) {
+		c.JSON(http.StatusOK, err.Error())
```

Comment:
i know err is not nil here, but it scares me to not check it lol

Reply:
ha, will throw that in there